database/data.sql

```diff
@@ -10,11 +10,11 @@
 --      ('Build projects', false),
 --      ('Get a job', false);
 
-INSERT INTO "posts" ("title", "subtitle", "body", "userId", "categoryId", "totalVotes", "views", "createdAt")
-VALUES ('Chomp''s Zero Sugar Beef Stick', 100, 'Only 100 calories, zero sugar and 10G of protein.', 1, 2, 12, 24, now());
+-- INSERT INTO "posts" ("title", "calories", "body", "userId", "categoryId", "totalVotes", "views", "createdAt")
+-- VALUES ('Chomp''s Zero Sugar Beef Stick', 100, 'Only 100 calories, zero sugar and 10G of protein.', 1, 2, 12, 24, now());
 
-insert into "users" ("username", "email", "password", "role", "createdAt")
-values ('markm', 'hello@gmail.com', 'password123', 'user', now());
+-- insert into "users" ("username", "email", "password", "role", "createdAt")
+-- values ('markm', 'hello@gmail.com', 'password123', 'user', now());
```

Comment:
You'll need to move this to the top of the file in case you ever run this again, since all the other tables need a userId as a reference, you need to create the user first